src/test/java/chat/tasks/TaskListTest.java

```diff
@@ -0,0 +1,30 @@
+package chat.tasks;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import chat.exceptions.ChatException;
+
+public class TaskListTest {
+
+    @Test
+    public void test1() {
```

Comment:
Might be good to name tests using the format stated in the Java Coding Standards. As quoted below: 

Underscores may be used in test method names using the following three part format `featureUnderTest_testScenario_expectedBehavior()`.